src/js/components/Search.jsx

```diff
@@ -6,16 +6,46 @@ import { SearchBar } from './SearchBar.jsx';
 export class Search extends PureComponent {
   constructor(props) {
     super(props);
-    this.state = { showResults: false };
+    this.state = {
+      columns: [{
+        key: 'name',
+        title: 'Name',
+      }, {
+        key: 'code',
+        title: 'Code',
+      }],
+      columnWidths: [400, 135],
+      showResults: false,
+    };
   }
 
-  // @todo remove forced condition
   // respond to search input
   handleSearchChange(data, searchTerm) {
     if (!data || !data.length) this.setState({ showResults: false });
     this.setState({ showResults: true, resultData: data, searchTerm });
   }
 
+  /**
+  * handleCalculateColumnWidths
+  *  - provides naïve, but sane, column widths
+  *  - currently, simply divides columns into equal widths
```

Comment:
doesn't work ❌. The real way to do it is to add support from the sussol-react-table API.  Most likely adding width/height properties to the Cell component using Blueprint's `renderCell` would be in order; though don't hold me to the speculation.